include/view.h

```diff
@@ -68,6 +69,7 @@ struct view_impl {
 	void (*move_to_back)(struct view *view);
 	struct view *(*get_root)(struct view *self);
 	void (*append_children)(struct view *self, struct wl_array *children);
+	bool (*is_related)(struct view *self, struct wlr_surface *surface);
```

Comment:
Suggest adding a short comment what exactly "is_related" means, same for the view_ wrapper function below.

Other than that: LGTM

Reply:
I added comments, thanks for the suggestion :+1: